larreco/ClusterFinder/Cluster3D_module.cc

```diff
@@ -480,18 +480,19 @@ namespace lar_cluster3d {
     /**
      *   Algorithm parameters
      */
-    bool m_enableMonitoring;      ///< Turn on monitoring of this algorithm
+    bool  m_onlyMakSpacePoints;   ///< If true we don't do the full cluster 3D processing
+    bool  m_enableMonitoring;     ///< Turn on monitoring of this algorithm
     float m_parallelHitsCosAng;   ///< Cut for PCA 3rd axis angle to X axis
     float m_parallelHitsTransWid; ///< Cut on transverse width of cluster (PCA 2nd eigenvalue)
 
     /**
      *   Tree variables for output
      */
     TTree* m_pRecoTree;            ///<
-    int m_run;                     ///<
-    int m_event;                   ///<
-    int m_hits;                    ///< Keeps track of the number of hits seen
-    int m_hits3D;                  ///< Keeps track of the number of 3D hits made
+    int   m_run;                   ///<
+    int   m_event;                 ///<
+    int   m_hits;                  ///< Keeps track of the number of hits seen
+    int   m_hits3D;                ///< Keeps track of the number of 3D hits made
```

Comment:
@SFBayLaser seet his block for instance.  The only changes are formatting.  This makes it difficult for us to find the real changes.

Reply:
Ah, I see... Ok, I suppose its challenging for me to find them too. I will try looking but will have to be tomorrow and I get pulled into a lost afternoon of meetings.